ibis/backends/dask/__init__.py

```diff
@@ -128,15 +128,6 @@ def _convert_schema(schema: sch.Schema):
     def _convert_object(cls, obj: dd.DataFrame) -> dd.DataFrame:
         return obj
 
-    def create_table(
-        self,
-        table_name: str,
-        obj: dd.DataFrame | None = None,
```

Comment:
Actually, I guess overriding this method made it so we could have different type annotations from super(). Should this not get removed so we keep the custom annotations?